pkg/controller/prometheus.go

```diff
@@ -161,20 +163,34 @@ func (c *Controller) checkAndCreatePrometheusAddlConfig(ctx context.Context, ten
 			return err
 		}
 	} else {
-		var scrapeConfigs []configmaps.ScrapeConfig
+		var scrapeConfigs, exceptedScrapeConfigs []configmaps.ScrapeConfig
```

Comment:
Can we change line 146 to include the namespace/tenant that is getting the Prometheus scrape configuration?